[2025-04-09T03:34:39.306+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-09T03:34:39.331+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: forex_ml_pipeline.create_s3_bucket scheduled__2025-03-30T12:00:00+00:00 [queued]>
[2025-04-09T03:34:39.342+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: forex_ml_pipeline.create_s3_bucket scheduled__2025-03-30T12:00:00+00:00 [queued]>
[2025-04-09T03:34:39.343+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 4
[2025-04-09T03:34:39.365+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): create_s3_bucket> on 2025-03-30 12:00:00+00:00
[2025-04-09T03:34:39.377+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'forex_ml_pipeline', 'create_s3_bucket', 'scheduled__2025-03-30T12:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/forex_ml_pipeline.py', '--cfg-path', '/tmp/tmpqk15uppx']
[2025-04-09T03:34:39.386+0000] {standard_task_runner.py:105} INFO - Job 6: Subtask create_s3_bucket
[2025-04-09T03:34:39.389+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=379) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-09T03:34:39.390+0000] {standard_task_runner.py:72} INFO - Started process 386 to run task
[2025-04-09T03:34:39.539+0000] {task_command.py:467} INFO - Running <TaskInstance: forex_ml_pipeline.create_s3_bucket scheduled__2025-03-30T12:00:00+00:00 [running]> on host ad9b7dc43921
[2025-04-09T03:34:39.723+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mikhail' AIRFLOW_CTX_DAG_ID='forex_ml_pipeline' AIRFLOW_CTX_TASK_ID='create_s3_bucket' AIRFLOW_CTX_EXECUTION_DATE='2025-03-30T12:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-30T12:00:00+00:00'
[2025-04-09T03:34:39.725+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-09T03:34:39.789+0000] {base.py:84} INFO - Retrieving connection 'my_s3_conn'
[2025-04-09T03:34:39.791+0000] {connection_wrapper.py:328} INFO - AWS Connection (conn_id='my_s3_conn', conn_type='aws') credentials retrieved from extra.
[2025-04-09T03:34:41.920+0000] {logging_mixin.py:190} INFO - Bucket 'usd-php-ml-pipeline-bucket' already exists. Skipping creation.
[2025-04-09T03:34:41.921+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-04-09T03:34:41.953+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-09T03:34:41.954+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=forex_ml_pipeline, task_id=create_s3_bucket, run_id=scheduled__2025-03-30T12:00:00+00:00, execution_date=20250330T120000, start_date=20250409T033439, end_date=20250409T033441
[2025-04-09T03:34:41.998+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-09T03:34:42.030+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-04-09T03:34:42.035+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
